spec/unit/provider/oneview_volume_attachment_synergy_spec.rb

```diff
@@ -13,6 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 ################################################################################
+require 'spec_helper'
 
 provider_class = Puppet::Type.type(:oneview_volume_attachment).provider(:synergy)
 
```

Comment:
Could move lines 20-22 to line ~27 so those do not get run unless we are actually running the tests (a.k.a. using api version >= 300)